### PR TITLE
Dev docker

### PR DESCRIPTION
**Pull request by AaronVasquez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +38,7 @@ brokernode# buffalo test
 ```
```
Comment (rfornea) [inline README.md]: Do we need some sort of database.dev.yml file as well?

Comment (AaronVasquez) [reply README.md]: This worked for me. The yml file in the `dev-dbs` branch looks to be the same as what's on master.

**Comment by AaronVasquez** [comment]
Actually, don't approve yet, it started failing locally :(

File: README.md (hunk 1 above)
Comment (rfornea) [reply README.md]: They are some differences.  Configuration for test is the same, but development and production are different.

Comment (AaronVasquez) [reply README.md]: still a little weird how the tests pass on travis, but not locally. Ill try copy and pasting the database.yml and see if that fixes it for me

**Pull request closed without merging**